CHANGES_NEXT_RELEASE

```diff
@@ -1,5 +1,8 @@
 - Fix: idPattern '.*' is now allowed in NGSIv2 registrations (#3458)
 - Add: Forwarded queries work for registrations using idPattern == '.*', but only for simple cases (#3458)
 - Add: Forwarded updates work for registrations using idPattern == '.*', but only for simple cases (#3458)
+- Add: CLI option -payloadMaxSize to let users decide the maximum allowed size of incoming payloads (#3492)
+- Add: CLI option -notifMaxSize to let users decide the maximum allowed size of the payload of notifications (#3492+)
+- Add: CLI option -outMsgMaxSize to let users decide the maximum allowed total size of any outgoing message (response, forward, and notification) (#3492+)
```

Comment:
Suggestio to make CLI flags more precise and homogeneous:

```
-reqPayloadMaxSize
-notifPayloadMaxSize
-outPayloadMaxSize
```

Internal artifacts (define, global variables, etc.) to be changed also accordingly.

Reply:
Amend: taking into account https://github.com/telefonicaid/fiware-orion/pull/3496#discussion_r288426746:

```
-inReqPayloadMaxSize
-notifPayloadMaxSize
-outReqPayloadMaxSize
```

(The introduction of -notifPayloadMaxSize depending of what we decide for https://github.com/telefonicaid/fiware-orion/pull/3496#discussion_r288425857)

Reply:
Sure, I never really liked the names.
However, outPayloadMaxSize is not correct.
The first two are about payload size while the last in the total message size (including HTTP Headers etc).

So ... -outTotalMaxSize ?

Reply:
If the size includes HTTP headers, etc. I think "Msg" is fine.

Reply:
Fixed in 636d28252   (with outTotalMaxSize)